src/util/errors.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+export const SERVER_EXTENSION_404 = (
```

Comment:
I would prefer renaming this to something hints to the fact this is a JSX element. I think names should describe the type of a value if it's not obvious. Perhaps `ERROR_JSX_404` instead?